classes/form/CustomerForm.php

```diff
@@ -275,4 +282,35 @@ private function validateByModules()
             }
         }
     }
+
+    /**
+     * Performs validation on field values.
+     * Adds error to the field object if value is not as expected.
+     */
+    private function validateFieldsValues(): void
+    {
+        $this->validateFieldIsCustomerName('firstname');
+        $this->validateFieldIsCustomerName('lastname');
+    }
+
+    /**
+     * Checks whether a field's value is a valid customer(person) name.
+     *
+     * @param string $fieldName
+     */
+    private function validateFieldIsCustomerName(string $fieldName): void
+    {
+        $field = $this->getField($fieldName);
+        if (null === $field) {
+            return;
+        }
+        $value = $field->getValue();
+        if (!empty($value) && false === (bool) Validate::isCustomerName($value)) {
+            $field->AddError($this->translator->trans(
```

Comment:
```suggestion
            $field->addError($this->translator->trans(
```

Just notice the method name is not the right one.